Sources/VLCMovieViewController.m

```diff
@@ -1891,4 +1925,56 @@ - (void) mediaMoreOptionsActionSheetDidToggleInterfaceLockWithState:(BOOL)state
 {
     [self toggleUILock];
 }
+
+#pragma mark - VLCMediaScrubProgressBarDelegate
+- (void)updateProgressScrubBarTimeLabel
+{
+    [_scrubProgressBar setRemainingTimeLabelText:[[_vpc remainingTime] stringValue]];
+    [_scrubProgressBar setElapsedTimeLabelText:[[_vpc playedTime] stringValue]];
+}
+
+- (void)updatePlaybackPosition
+{
+    if(!_positionSet) {
+        float position = _scrubProgressBar.progressSlider.value;
+        [_vpc setPlaybackPosition:position];
+        [_vpc setNeedsMetadataUpdate];
```

Comment:
can we move the setNeedsMetadataupdate into the setPlaybackPosition and remove it actually from the header of VLCPlaybackController? I don't think this should be done from the movieviewcontroller in general